compiler/rustc_expand/src/base.rs

```diff
@@ -979,12 +979,6 @@ impl SyntaxExtension {
         let stability = find_attr!(attrs, AttributeKind::Stability { stability, .. } => *stability);
 
         // FIXME(jdonszelmann): make it impossible to miss the or_else in the typesystem
```

Comment:
I forgot what my own fixme here refers to tbh. I think it looks like its already handled? Could you remove it?